CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 0.7.7
+- Added support for retrieving the Pubnub subscription details
```

Comment:
Does pubnub require us to provide an endpoint for receiving update events?

How does the pubnub data get to us?